src/applications/simple-forms/26-4555/definitions/customFullNameUI.js

```diff
@@ -0,0 +1,39 @@
+/* fullNameUI.js
+VA-Product-Forms team's custom version of /platform/forms-system/src/js/definitions//fullName.
+This one's a function that takes an optional custom-labels object param.
+*/
+const customUiSchema = ({
+  firstNameLabel = 'First name',
+  middleNameLabel = 'Middle name',
+  lastNameLabel = 'Last name',
```

Comment:
@tlei123, are we able to drop this code and just import the platform code now that we aren't saying "Your first name" etc.? I think their definition uses "First name" etc,

Reply:
Yeah they're already using those labels from what I can tell: https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/definitions/fullName.js

Reply:
customFullNameUI now deleted.